src/web_server_config.cpp

```diff
@@ -91,7 +55,7 @@ handleConfigPost(MongooseHttpServerRequest *request, MongooseHttpServerResponseS
     bool config_modified = web_server_config_deserialise(doc, storage.equals("factory"));
 
     StaticJsonDocument<128> reply;
-    reply["config_version"] = config_version;
+    reply["config_version"] = doc["config_version"];
```

Comment:
there's no config_version in /config and so no config_version should then be posted too in handleConfigPost().
doc["config_version"] won't exist, jsut keep it like it was:
reply["config_version"] = config_version;

Reply:
So I did that initially because `config_version ` was internal and not exposed to I needed to pass it through the document.
Now that `config_version`  is exposed I can clean yes.